### PR TITLE
enhance: Add http method to control datacoord garbage collection

### PR DESCRIPTION
**Pull request by congqixia**
See also #29051

### CODE REVIEW

File: internal/datacoord/server.go [hunk 1]
```diff
@@ -405,6 +407,44 @@ func (s *Server) startDataCoord() {
 		s.compactionViewManager.Start()
 	}
 	s.startServerLoop()
+
+	http.Register(&http.Handler{
+		Path: "/datacoord/garbage_collection/pause",
```
Comment (xiaofan-luan) [inline internal/datacoord/server.go:412]: this path seems to be not a normal path?

**Comment by wayblink** [comment]
@congqixia @xiaofan-luan  Can we expose it in proxy? So backup tool can use it with proxy address only. Or We have to get and config datacoord address to use it.

**Pull request merged**